src/openforms/forms/tests/test_serializers.py

```diff
@@ -83,6 +89,33 @@ def test_date_variable_with_non_trivial_json_trigger(self):
 
         self.assertTrue(serializer.is_valid())
 
+    def test_registration_backend_action_errors_no_value(self):
```

Comment:
test name makes me think this should be a validation error because the `value` is missing, but in fact what is being tested is that a valid `type` is specified. I think we need both tests?